tools/release.js

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env node
+'use strict';
+
+const yargs = require('yargs/yargs');
+const { hideBin } = require('yargs/helpers');
+const readline = require('readline').createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+const { buildDockerImages, pushDockerImage } = require('./docker-utils');
+
+async function main() {
+  const argv = yargs(hideBin(process.argv))
```

Comment:
I'm not a huge fan of the `yargs` developer experience, but it's the best CLI tool I know. It would obviously be possible to build this functionality without packaging it as a CLI, but I think it turned out pretty good. This way, there is a single entrypoint that yells at you if you use it wrong.